docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -6,14 +6,13 @@ Via a simple `fit()` call, AutoGluon can produce highly-accurate models to predi
 To start, import autogluon.tabular and TabularPrediction module as your task:
```

Comment:
"TabularPrediction" no longer exists, update to TabularPredictor

Reply:
Added